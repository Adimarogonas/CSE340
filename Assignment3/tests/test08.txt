{
    s = + e * f g;
    t = ! < e s;
    x = y;
}
