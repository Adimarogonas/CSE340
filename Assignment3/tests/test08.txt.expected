s, e, f, g: ? #
t: bool #
x, y: ? #
